imports/components/@primitives/layout/global/index.js

```diff
@@ -160,50 +173,52 @@ class GlobalWithoutData extends Component {
       "/stories/",
     ];
 
-    if (Meteor.isCordova) {
-      document.addEventListener("click", linkListener);
-      document.addEventListener("deviceready", () => {
-        universalLinks.subscribe(null, ({ path }) => {
-          this.setState({ universalLinkLoading: true });
-          let isQueryRoute = false;
-          queryRoutes.forEach((url) => {
-            if (path.includes(url)) isQueryRoute = true;
-          });
-          if (isQueryRoute) {
-            // const pathArray = path.split("/").filter(Boolean);
-            // const section = pathArray[0];
-            // const urlTitle = pathArray[1];
-            // this.props.client.query({ query, variables: { urlTitle })
-            //   .then(({ data: { somehwere: id } }) => {
-            //     this.go(`/#{this.section}/);
-            //   })
-            // this.go(`/${section}`);
-            alert("this is a query route");
-            return;
-          }
-          this.go(path);
-        });
-      }, false);
+    this.state = ({ universalLinkLoading: true });
+    let isQueryRoute = false;
+
+    queryRoutes.forEach((url) => {
+      if (path.includes(url)) isQueryRoute = true;
+    });
+
+    if (isQueryRoute) {
+      // const pathArray = path.split("/").filter(Boolean);
+
+      // const section = pathArray[0];
+      // let urlTitle = pathArray[1];
+
+      // if (pathArray.length === 2) {
+        // const parent = pathArray[1];
+        // urlTitle = pathArray[2];
+      // }
+
+      // this.props.client.query({ query, variables: { urlTitle })
+      //   .then(({ data: { somehwere: id } }) => {
+      //     this.go(`/#{this.section}/);
+      //   })
+      // this.go(`/${section}`);
+      alert("this is a query route");
+      return;
     }
-  }
 
-  componentWillUnMount() {
-    if (Meteor.isCordova) universalLinks.unsubscribe(null);
+    // XXX accounts for watch and read
+    if (path === "/watchandread") this.go("/");
```

Comment:
this needs to either be a return after the function, or a switch statement. Otherwise `this.go` will be fired twice